packages/synchro-charts/src/components/sc-dial/sc-dial-base/sc-dial-base.css

```diff
@@ -0,0 +1,46 @@
+sc-dial-base .error {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  font-size: var(--font-size-1);
+  line-height: var(--line-height-3);
+}
+
+sc-dial-base .error .warning-symbol {
+  transform: scale(0.7);
+}
+
+sc-dial-base .error sc-error-badge div {
```

Comment:
this will be very flaky, if someone alters the DOM at all it'll break, prefer to just give it it's own class name